src/device/usbd.h

```diff
@@ -377,7 +377,7 @@ TU_ATTR_WEAK bool tud_vendor_control_complete_cb(uint8_t rhport, tusb_control_re
   #define TUD_RNDIS_ITF_PROTOCOL 0xFF /* CDC_COMM_PROTOCOL_MICROSOFT_RNDIS */
 #else
   /* Windows 7+ */
-  #define TUD_RNDIS_ITF_CLASS    0xE0
+  #define TUD_RNDIS_ITF_CLASS    TUSB_CLASS_WIRELESS_CONTROLLER
```

Comment:
@majbthrd  First it is not related to this 380 line, but on line 350 `#define TUD_CDC_ECM_DESCRIPTOR()` , github doesn't allow to comment on unmodified line !!!. I just notice ECM doesn't have Interface Assoc Descriptor. It is a bit awkward since it is an CDC subclass. It could work without IAD, but maybe since our example has only 1 usbnet, it could run into issue with a composite device. I will try to add IAD and test. Though just want to check with you first

Reply:
update: hehe, I couldn't wait until you wake up, give it a try by adding IAD and it works fine on Linux. I will include this in my next push with IAD enhance, then ALT ITF. Let me know if there is anything preventing us to use IAD in this case. 

```
  /* Interface Association */\
  8, TUSB_DESC_INTERFACE_ASSOCIATION, _itfnum, 2, TUSB_CLASS_CDC, CDC_COMM_SUBCLASS_ETHERNET_NETWORKING_CONTROL_MODEL, 0, 0,\
```